libs/pages/dashboard/src/lib/pages-dashboard.tsx

```diff
@@ -0,0 +1,157 @@
+import { Fragment, useCallback, useEffect, useState } from 'react';
+import { OrderModel, OrderStatus } from '@pasnik/api/data-transfer';
+import { useOrdersFacade } from '@pasnik/orders-data-access';
+import { useHistory } from 'react-router-dom';
+import { Tab } from '@headlessui/react';
+import OrderList from './order-list/order-list';
+import { BeakerIcon } from '@heroicons/react/outline';
+import classnames from 'classnames';
+
+/* eslint-disable-next-line */
+export interface PagesDashboardProps {}
+
+export function PagesDashboard(_: PagesDashboardProps) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [currentTab, setCurrentTab] = useState(0);
+  const [activeOrders, setActiveOrders] = useState<OrderModel[]>([]);
+  const [completedOrders, setCompletedOrders] = useState<OrderModel[]>([]);
+  const { fetchOrders } = useOrdersFacade();
+  const history = useHistory();
+
+  useEffect(() => {
+    fetchOrders().then((fetchedOrders) => {
+      const [active, completed] = fetchedOrders.reduce(
+        (acc, cur) => {
+          if (cur.status === OrderStatus.InProgress) {
+            acc[0].push(cur);
+          } else {
+            acc[1].push(cur);
+          }
+          return acc;
+        },
+        [[], []] as Array<OrderModel[]>
+      );
+      setActiveOrders(active);
+      setCompletedOrders(completed);
+      setIsLoading(false);
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const tabs = [
+    { name: 'Aktywne', count: activeOrders.length },
+    { name: 'Zakończone', count: completedOrders.length },
+  ];
+
+  const handleTabGroupChange = useCallback((index) => {
+    setCurrentTab(index);
+  }, []);
+
+  const createOrderHandler = useCallback(() => {
+    history.push('/create-order');
+  }, [history]);
+
+  return (
```

Comment:
side note, divide this component to smaller ones

Reply:
Done